### ISSUE TITLE
Customizing <Icon> font URL in ejected CRA app

### ISSUE THREAD

**Issue by JesperWe** [ant-design/ant-design#5538]
#### Environment(required)

- antd version: 2.8.3
- OS and its version: Ubuntu 14
- Browser and its version: Chrome

#### What did you do? Please provide steps to re-produce your problem.

Customized theme according to https://ant.design/docs/react/use-with-create-react-app#Customize-Theme, specifying `@icon-url`: 

`loader: 'style!css!postcss!less?{modifyVars:{"icon-url":"https://hostname.com/iconfilename"}}'`

#### What do you expected?

Resulting client code loads my custom icon font.

#### What happened?

Standard AntDesign font still loaded, and a weird segment appears in client `<head>`;

`<style type="text/css">@icon-url : https:/hostname.com/iconfilename;
</style>`

This only happens with `@icon-url`, I have plenty of other customized variables which work fine.

I can't tell if this is a webpack, less or antd issue, it seems all three are doing some processing of this variable. Notice how // has become / in the URL.  Any advice on how to find the cause...?

**Comment by afc163** [comment]
```diff
- loader: 'style!css!postcss!less?{modifyVars:{"icon-url":"https://hostname.com/iconfilename"}}'
+ loader: 'style!css!postcss!less?{modifyVars:{"icon-url":"'https://hostname.com/iconfilename'"}}'
```

It is mentioned at https://ant.design/docs/react/customize-theme#1)-Using-theme-property-(recommended-way), the Note part.

**Comment by JesperWe** [comment]
Still does not work.

webpack.config:
```
{
	test: /\.less$/,
	loader: 'style!css!postcss!less?{modifyVars:{"font-size-base":"15px","primary-color":"#56c8be","text-color-secondary":"#b00843","screen-xs":"480px","screen-sm":"586px","screen-md":"768px","screen-lg":"992px","icon-url":"\'https://hostname/treatyaicon\'","font-family":"\'Noto Sans\', Helvetica, Arial, sans-serif;","rate-star-color":"#b00843","border-radius-base":"0"}}'
},
```

Compile result (Notice https:// becomes https:/ which does not parse as a URL):
```
Error in ./~/antd/lib/style/index.less
Module not found: ./https:/hostname.com/treatyaicon.eot in /home/treat/treatya/tyc/node_modules/antd/lib/style

 @ ./~/css-loader!./~/postcss-loader!./~/less-loader?{modifyVars:{"font-size-base":"15px","primary-color":"#56c8be","text-color-secondary":"#b00843","screen-xs":"480px","screen-sm":"586px","screen-md":"768px","screen-lg":"992px","icon-url":"'https:/hostname/treatyaicon'","font-family":"'Noto Sans', Helvetica, Arial, sans-serif;","rate-star-color":"#b00843","border-radius-base":"0"}}!./~/antd/lib/style/index.less 6:10492-10543 6:10578-10629
```

**Comment by JesperWe** [comment]
OK, this turned out to be a Webpack bug. It is fixed in the version used by CreateReactApp v0.9.5.
